sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q14a.sf100/explain.txt

```diff
@@ -713,13 +713,13 @@ Input [6]: [i_brand_id#8, i_class_id#9, i_category_id#10, sales#80, number_sales
 Condition : (isnotnull(sum(CheckOverflow((promote_precision(cast(cast(ws_quantity#68 as decimal(10,0)) as decimal(12,2))) * promote_precision(cast(ws_list_price#69 as decimal(12,2)))), DecimalType(18,2), true))#82) AND (cast(sum(CheckOverflow((promote_precision(cast(cast(ws_quantity#68 as decimal(10,0)) as decimal(12,2))) * promote_precision(cast(ws_list_price#69 as decimal(12,2)))), DecimalType(18,2), true))#82 as decimal(32,6)) > cast(ReusedSubquery Subquery scalar-subquery#49, [id=#50] as decimal(32,6))))
 
 (128) Project [codegen id : 135]
-Output [6]: [sales#80, number_sales#81, web AS channel#83, i_brand_id#8, i_class_id#9, i_category_id#10]
+Output [10]: [web AS channel#83, i_brand_id#8, i_class_id#9, i_category_id#10, sales#80, number_sales#81, web AS channel#83, i_brand_id#8, i_class_id#9, i_category_id#10]
 Input [6]: [i_brand_id#8, i_class_id#9, i_category_id#10, sales#80, number_sales#81, sum(CheckOverflow((promote_precision(cast(cast(ws_quantity#68 as decimal(10,0)) as decimal(12,2))) * promote_precision(cast(ws_list_price#69 as decimal(12,2)))), DecimalType(18,2), true))#82]
 
 (129) Union
 
 (130) Expand [codegen id : 136]
-Input [6]: [sales#46, number_sales#47, channel#51, i_brand_id#8, i_class_id#9, i_category_id#10]
+Input [10]: [channel#51, i_brand_id#8, i_class_id#9, i_category_id#10, sales#46, number_sales#47, channel#51, i_brand_id#8, i_class_id#9, i_category_id#10]
```

Comment:
Ur, is this expected? It looks like duplicating inputs. Maybe, a regression?

Reply:
Yes. Only move simplify expression rules before `// Operator push down`.